policy/policy.sample.json

```diff
@@ -0,0 +1,70 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1", # names can be same for multiple tenants thus use uuid.
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants.
+            "Associations": [ # can attach multiple tenants associations in case you want blanket policy support.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
```

Comment:
So this specifies the specific segment on which the policy should be defined, right?

And I assume that our permission system would have to prevent tenant-A from making associations involving a segment of tenant-B. Only admins/root/system accounts could create policies with associations that stretch across tenants?

If that's so, I assume that an unprivileged tenant should be able to drop the 'tenant' part of the association, since if it's missing we can just full the 'current tenant'.